src/fsharp/tast.fs

```diff
@@ -273,13 +273,10 @@ type TyparKind =
       | TyparKind.Type -> None
       | TyparKind.Measure -> Some "Measure"
 
-    [<DebuggerBrowsable(DebuggerBrowsableState.Never)>]
-    member x.DebugText  =  x.ToString()
+    //[<DebuggerBrowsable(DebuggerBrowsableState.Never)>]
+    //member x.DebugText  =  x.ToString()
 
-    override x.ToString() = 
```

Comment:
This is the change that seems to break tests.
the casing of 'type' has changed.